topologies/available_topo.yml

```diff
@@ -1,32 +1,37 @@
+# topology_default - key used to set the default topology to use
+#  - If tag has no valid value or is missing, the first entry in topologies will be used as the default
+#
+# topologies - array of topologies to use
+# - Topology_Name: Must be less than 50 characters. Anything larger will be truncated
+# - Topology_File: Must include path to yml file in topologies directory
```

Comment:
If the path is invalid the entire topology entry is ignored.

Reply:
Corrected